koin-projects/koin-test/build.gradle

```diff
@@ -7,7 +7,7 @@ dependencies {
     // Koin
     compile project(":koin-core")
     compile "junit:junit:$junit_version"
-    testImplementation "org.mockito:mockito-inline:$mockito_version"
+    compile "org.mockito:mockito-inline:$mockito_version"
```

Comment:
Forcing mockito is a too strong requirement IMHO. There are project that use different mocking frameworks - which might conflict - or do not want any mocking framework at all - for example due to build speed considerations.

Reply:
Yes, I clearly need a way to mock needed injected params, as we don't need to write a parameter creator for each of them. It will help people scale their module with that.

Another thigns can be to backport just the mock part of a lib

Reply:
I finally rework to use the `MockProvider` rule, that allows to provide mock with the desired framework

Reply:
Much better. Still not the greatest fan of mocking by default (via `?: MockParameter()`) but at least you have to opt-in or you would get an error.